src/http/formdata.cr

```diff
@@ -133,7 +133,7 @@ module HTTP::FormData
 
     parts = content_disposition.split(';')
     type = parts[0]
-    raise Error.new("Invalid Content-Disposition: not form-data") unless type == "form-data"
+  raise Error.new("Invalid Content-Disposition: not form-data or file") unless ["form-data", "file"].includes?(type)
```

Comment:
I think you incorrectly removed 2 spaces at the beginning.

Also, you should use a Tuple instead of an array to avoid having to allocate the array, so something like: `{"form-data", "file"}.includes? type`.

Reply:
thanks, will update